src/MensaWidget/MWDockingBarTest.class.st

```diff
@@ -43,6 +43,22 @@ MWDockingBarTest >> tearDown [
 	self dockingbar: nil
 ]
 
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'LC 7/5/2024 00:56'
+}
+MWDockingBarTest >> testOpenSettings [
+
+	| menu submenu widget |
+	
+	TheWorldMainDockingBar showMensaMenuInWorldMainDockingBar: true.
+	self dockingbar allMorphsDo: [:each | (each name asString includesSubstring: 'MensaMenu') ifTrue: [menu := each]].
```

Comment:
line indent

Reply:
why exactly would this be indented?

Reply:
its tbh too long for one line

Reply:
ah, i thought Tom just wanted to indent the self dockingbar line

Reply:
I wrote a helper function for that functionality i think that makes the code easier to read